tests/Tests/100__deploy/100__installation/102__post_install.robot

```diff
@@ -329,3 +341,14 @@ Verify Requests Contains Expected Values
     [Arguments]   ${cpu}  ${memory}  ${requests}
     Should Be Equal As Strings    ${requests['cpu']}  ${cpu}
     Should Be Equal As Strings    ${requests['memory']}  ${memory}
+
+Fetch Cluster Worker Nodes Info
+    [Documentation]    Fetch information about the nodes of the cluster
+    ...    Args:
+    ...        None
+    ...    Returns:
+    ...        cluster_nodes_info(list(dict)): Cluster nodes information
+    @{cluster_nodes_info}=    Oc Get    kind=Node    api_version=v1
+    ...    label_selector=node-role.kubernetes.io/worker=,node-role.kubernetes.io!=master,node-role.kubernetes.io!=infra
+    [Return]    @{cluster_nodes_info}
```

Comment:
this can be useful as common keyword, maybe in a dedicated resource file or in ODS.robot

Reply:
Done